src/rgw/rgw_sal.h

```diff
@@ -1536,26 +1536,59 @@ class LuaScriptManager {
  */
 class StoreManager {
 public:
+  struct Config {
+    /** Name of store to create */
+    std::string store_name;
+    /** Name of filter to create or "none" */
+    std::string filter_name;
```

Comment:
Does it mean we can configure only single filter? I assumed we can have multiple filter layers on top of a store.

Reply:
For then moment, yes.  But one of the points of this work is to move config into an opaque struct, so that we can change things like this transparently.  Eventually, we'll need a policy engine to determine filters.